Try Catch Finally Throw

**1、throw:** 当问题出现时，程序会抛出一个异常。这是通过使用 **throw** 关键字来完成的。

**2、try:** **try** 块中的代码标识将被激活的特定异常。它后面通常跟着一个或多个 catch 块。

**3、catch:** 在您想要处理问题的地方，通过异常处理程序捕获异常。**catch** 关键字用于捕获异常。

**4、finally：** 关键字finally放在catch之后，如果异常没有被catch捕获，会使用关键字去清理释放资源